spec/classes/init_spec.rb

```diff
@@ -255,6 +257,7 @@
     let(:facts) {{
       :operatingsystem => 'Fedora',
       :operatingsystemmajrelease => 20,
```

Comment:
is operatingsystemmajrelease no longer used? if so can you remove it to prevent confusion?